lib/rdoc/code_object/method_attr.rb

```diff
@@ -63,12 +63,6 @@ class RDoc::MethodAttr < RDoc::CodeObject
 
   attr_reader :arglists
 
-  ##
-  # Pretty parameter list for this method
-
-  attr_reader :param_seq
```

Comment:
It seems that `AnyMethod#arglists` uses `param_seq`: https://github.com/ruby/rdoc/blob/fa0a12bf2709e0d3ba4fa24d67962d08e4c223fe/lib/rdoc/code_object/any_method.rb#L88-L89

Can we remove the clause too?

Reply:
No because `AnyMethod` defines its own `param_seq` method:

https://github.com/ruby/rdoc/blob/a253a8d163c36ede4064083d5d43f69f789d0c51/lib/rdoc/code_object/any_method.rb#L278

Reply:
Oh, sorry.

Reply:
No worries 🙂 The codebase is pretty confusing now and that's why we need to refactor it first.
Please keep raising any concern you have as it's easy for me to get lost when looking at it too 😅